### PR TITLE
Doc: OpenPype Versioning

### PR DESCRIPTION
**Pull request by ClementHector**
A short explanation of the versioning system in OpenPype.

### CODE REVIEW

File: website/docs/admin_distribute.md [hunk 1]
```diff
@@ -63,4 +63,18 @@ You can run OpenPype with `--use-staging` argument to add use staging versions.
...
-:::
\ No newline at end of file
+:::
+
+### OpenPype versionning
+
+Openpype version control is based on sementic versioning
+
+:::note
+The version of openpye is indicated by the variable `__version__` in the file `.\openpype\version.py`
```
Comment (BigRoy) [inline website/docs/admin_distribute.md:73]: `openpye` -> `openpype`

File: website/docs/admin_distribute.md [hunk 2]
```diff
@@ -63,4 +63,18 @@ You can run OpenPype with `--use-staging` argument to add use staging versions.
...
+:::
+
+For example OpenPype will consider the versions in this order: `3.8.0-nightly` < `3.8.0-nightly.1` < `3.8.0-rc.1` < `3.8.0` < `3.8.1-nightly.1` <`3.8.1` < `3.9.0` < `3.10.0` < `4.0.0`
+
+See https://semver.org/ for more details
+
+For studios customizing the source code of Openpype, it is recommended to build by adding a name and a number after the PATCH and not to deploy 3.8.0 from original OpenPype repositoy. For example, your builds will be `3.8.0-yourstudio.1` < `3.8.0-yourstudio.2` < `3.8.1-yourstudio.1`
```
Comment (BigRoy) [inline website/docs/admin_distribute.md:80]: `repositoy` -> `repository`

Plus, missing period at the end of the sentence.

**Review by BigRoy — CHANGES_REQUESTED** [review]
Additional to the comments, I'd make sure to use OpenPype as name (with both the O and P as a capital letter) for consistency.

**Review by BigRoy — APPROVED** [review]
Only note I'd have is to end the sentences with a period (`.`). 

Other than that, I'll leave this for further review by someone from the OpenPype team. Probably @mkolar would be a good fit to check?

**Review by mkolar — APPROVED** [review]

File: website/docs/admin_distribute.md [hunk 3]
```diff
@@ -63,4 +63,18 @@ You can run OpenPype with `--use-staging` argument to add use staging versions.
 
 :::note
 Running staging version is identified by orange **P** icon in system tray.
-:::
\ No newline at end of file
+:::
+
+### OpenPype versioning
+
+OpenPype version control is based on semantic versioning
```
Comment (mkolar) [inline website/docs/admin_distribute.md:70]:
```suggestion
OpenPype version control is based on semantic versioning.
```

File: website/docs/admin_distribute.md [hunk 4]
```diff
@@ -63,4 +63,18 @@ You can run OpenPype with `--use-staging` argument to add use staging versions.
...
+
+:::note
+The version of OpenPype is indicated by the variable `__version__` in the file `.\openpype\version.py`
```
Comment (mkolar) [inline website/docs/admin_distribute.md:73]:
```suggestion
The version of OpenPype is indicated by the variable `__version__` in the file `.\openpype\version.py`.
```

File: website/docs/admin_distribute.md [hunk 5]
```diff
@@ -63,4 +63,18 @@ You can run OpenPype with `--use-staging` argument to add use staging versions.
...
+:::
+
+For example OpenPype will consider the versions in this order: `3.8.0-nightly` < `3.8.0-nightly.1` < `3.8.0-rc.1` < `3.8.0` < `3.8.1-nightly.1` <`3.8.1` < `3.9.0` < `3.10.0` < `4.0.0`
+
+See https://semver.org/ for more details
```
Comment (mkolar) [inline website/docs/admin_distribute.md:78]:
```suggestion
See https://semver.org/ for more details.
```

File: website/docs/admin_distribute.md [hunk 6]
```diff
@@ -63,4 +63,18 @@ You can run OpenPype with `--use-staging` argument to add use staging versions.
...
+
+For studios customizing the source code of OpenPype, a practical approach could be to build by adding a name and a number after the PATCH and not to deploy 3.8.0 from original OpenPype repository. For example, your builds will be: `3.8.0-yourstudio.1` < `3.8.0-yourstudio.2` < `3.8.1-yourstudio.1`
```
Comment (mkolar) [inline website/docs/admin_distribute.md:80]:
```suggestion
For studios customizing the source code of OpenPype, a practical approach could be to build by adding a name and a number after the PATCH and not to deploy 3.8.0 from original OpenPype repository. For example, your builds will be: `3.8.0-yourstudio.1` < `3.8.0-yourstudio.2` < `3.8.1-yourstudio.1`.
```

File: website/docs/admin_distribute.md [hunk 7]
```diff
@@ -63,4 +63,18 @@ You can run OpenPype with `--use-staging` argument to add use staging versions.
...
+
+### OpenPype versioning
+
+OpenPype version control is based on semantic versioning
+
+:::note
+The version of OpenPype is indicated by the variable `__version__` in the file `.\openpype\version.py`
+:::
+
+For example OpenPype will consider the versions in this order: `3.8.0-nightly` < `3.8.0-nightly.1` < `3.8.0-rc.1` < `3.8.0` < `3.8.1-nightly.1` <`3.8.1` < `3.9.0` < `3.10.0` < `4.0.0`
```
Comment (mkolar) [inline website/docs/admin_distribute.md:76]:
```suggestion
For example OpenPype will consider the versions in this order: `3.8.0-nightly` < `3.8.0-nightly.1` < `3.8.0-rc.1` < `3.8.0` < `3.8.1-nightly.1` <`3.8.1` < `3.9.0` < `3.10.0` < `4.0.0`.
```

**Comment by mkolar** [comment]
Apart from the full stops :)

**Comment by mkolar** [comment]
thank you

**Pull request merged**